config/opensearch_dashboards.yml

```diff
@@ -76,6 +76,13 @@
 # must be a positive integer.
 #opensearch.requestTimeout: 30000
 
+
+# Set the value of this setting to true to enable memory circuit breaker for large query response
```

Comment:
I think this can be cleaned up to something like: `Enables the memory circuit breaker that prevents heap out of memory errors for large query responses.`

It would be great to have this be comprehensive documentation so that engineers modifying this file don't have to search around for more information, as mentioned in https://github.com/opensearch-project/documentation-website/issues/214

Reply:
I have submitted a request here : https://github.com/opensearch-project/documentation-website/issues/470 
This is to add more documentation regarding the new introduced `memoryCircuitBreaker` in `ClientOptions` interface. OSD as a consumer of opensearch-js client. Will take a look how we can add more document over there

Reply:
I think a stretch goal would be to start documentation for all of the .yml options as mentioned in https://github.com/opensearch-project/documentation-website/issues/214. We may not be able to do that now, but let's at least add as much relevant documentation to the .yml file itself for now so that when we do add the documentation for all options it's all there.